src/main/java/edu/harvard/iq/dataverse/Dataverse.java

```diff
@@ -322,8 +322,31 @@ public boolean isHarvested() {
         return harvestingClient != null; 
     }
     */
-    
-    
+    private boolean metadataBlockFacetRoot;
+
+    public boolean isMetadataBlockFacetRoot() {
+        return metadataBlockFacetRoot;
+    }
+
+    public void setMetadataBlockFacetRoot(boolean metadataBlockFacetRoot) {
```

Comment:
Should this be set to false when you create a new Dataverse? Also for the root dataverse do we need to set it to true - since it doesn't have an owner? And it seems as if the only way to get a child dataverse to not inherit facets from its parent would be to give it its own facets via the api, correct?

Reply:
It should be `false` by default. Because it is a boolean primitive, the default value is `false`. I was following the same pattern as with `metadataBlockRoot` and `facetRoot`.

As for the root dataverse, technically, there is no need to set it to `true` because of the way `getMetadataBlockFacetRoot` was coded (checking for the owner). But if `metadataBlockRoot` and others are set to `true` in the root dataverse, then I guess it should be.

Reply:
@adaybujeda - yes, sorry I was away for a few days. I just want to do another pass through the code, but I will probably send it along to QA shortly.